redisson-all/pom.xml

```diff
@@ -162,6 +162,22 @@
             <version>[1.4,)</version>
             <scope>provided</scope>
         </dependency>
+	    <dependency>
+            <groupId>net.bytebuddy</groupId>
+            <artifactId>byte-buddy</artifactId>
+            <version>1.6.14</version>
```

Comment:
isn't version inherited from the parent pom (can be omitted)? I would also explicitly set 
```code
<optional>false</optional>
```

Reply:
dependencies are not defined in parent pom, they are in redisson module so we have to explicitly define the versions.  I will add the `<optional>false</optional>` tag

Reply:
maybe the versions could be defined as properties then in the parent pom to avoid having to maintain versions in more than one place, wdyt?